src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoQueryModel.cs

```diff
@@ -356,6 +357,27 @@ public override  IEnumerable<SqlSmoObject> Query(SmoQueryContext context, string
                     return ret;
                 }
             }
+            Schema parentSchema = context.Parent as Schema;
+            if (parentSchema != null)
+            {
+                List<NodePropertyFilter> filters = new List<NodePropertyFilter>();
+                filters.Add(new NodePropertyFilter()
+                {
+                    Property = nameof(Schema),
+                    Type = typeof(string),
+                    Values = new List<object> { parentSchema.Name },
+                    ValidFor = ValidForFlag.All 
+                });
+                filter = INodeFilter.AddPropertyFilterToFilterString(filter, filters, this.GetType(), context.ValidFor);
+                var retValue = parentSchema.Parent.Tables;
+                if (retValue != null)
+                {
+                    retValue.ClearAndInitialize(filter, extraProperties);
+                    var ret = new SmoCollectionWrapper<Table>(retValue).Where(c => PassesFinalFilters(parentSchema, c));
+                    Logger.Verbose("End query Table");
```

Comment:
this log message seems to not have enough context to be useful.  Maybe at least include something about SmoQueryModel or ObjectExplorer, or doesn't the Logger automatically capture some context.